src/hooks/useOverlayManager.tsx

```diff
@@ -62,5 +79,27 @@ export const useOverlayManager = () => {
     setOverlayOpen(false)
   }, [setOverlayOpen])
 
-  return { handleOverlayOpen, handleOverlayClose }
+  const handleOverlayContainerOpen = useCallback(() => {
+    if (overlayContainerRef.current === null) {
+      return
+    }
+    overlayContainerRef.current.style.visibility = 'visible'
```

Comment:
We shouldn't directly operate on `overlayContainerRef` style, this way it's easy to override styles applied by other components/libraries etc. It's a component that we render ourselves so we can easily do that styling by applying conditional styles.